zinc/src/test/scala/sbt/inc/ReporterTest.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Zinc - The incremental compiler for Scala.
+ * Copyright Lightbend, Inc. and Mark Harrah
+ *
+ * Licensed under Apache License 2.0
+ * (http://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package sbt.inc
+
+import sbt.internal.inc.ZincBuildInfo._
+import sbt.internal.inc.{ BridgeProviderSpec, StringVirtualFile }
+import sbt.io.IO.{ withTemporaryDirectory => withTmpDir }
+import scala.collection.JavaConverters._
+import xsbti.Action
+
+object ReporterTest extends BridgeProviderSpec {
```

Comment:
This is a test demonstrating the use of new code action in Scala 2.13.12-bin-286f8d7